app/views/admins/_user_entry.haml

```diff
@@ -37,6 +37,11 @@
           %dd= user.username
           %dt= t('.email')
           %dd= user.email
+          %dt= t('.role.label')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.